source-map.bs

```diff
@@ -63,6 +63,12 @@ spec:bikeshed-1; type:dfn; for:railroad; text:optional
     "publisher": "W3C",
     "status": "Living Standard",
     "title": "WebAssembly custom section"
+  },
+  "varuint32": {
+    "href": "https://github.com/WebAssembly/design/blob/main/BinaryEncoding.md#varuintn",
```

Comment:
This is the wrong thing to reference. It needs to be some part of the WebAssembly binary format spec. https://webassembly.github.io/spec/core/binary/index.html

Reply:
I think you want to refer to `u32` in https://webassembly.github.io/spec/core/binary/values.html#integers

Reply:
Actually you should refer directly to the `name` encoding at https://webassembly.github.io/spec/core/binary/values.html#names which already is based on vec (which starts with its length) and the UTF-8 encoding.

Reply:
Done